site/_wiki/Documentation/ConfigurationGuide.md

```diff
@@ -0,0 +1,418 @@
+
+> The manufacturers digitizer dimensions can often times be slightly wrong
+  (usually rounded up), so getting this right ensures that users can keep using
+  the exact same tablet area between different tablets in OpenTabletDriver
+
+#### Getting maximum X and Y values
+
+You can get the X and Y values from OpenTabletDriver's tablet debugger.
+Move your tool (e.g. pen) to the outer edges (usually right side and bottom
+side) of the digitizer area while keeping an eye on the reported X and Y values
+in the tablet debugger.
+
+If your tablet is reporting very quickly, it might be worth using the tablet data recording feature.
+> A way to easily display the maximum value returned is planned but not
+  implemented yet. See [OpenTabletDriver#2433] for more information.
+
+The `MaxX` and `MaxY` values should be equal to the highest observed value
+reported by the tablet on each axis.
+
+To ensure getting the highest possible X and Y values, make sure that you're
+moving your tool to the edge at multiple angles.
+
+If your achieved `MaxX` and `MaxY` values seem incorrect (such as 4096 x 4096),
+you might be initializing the tablet incorrectly, or you're not using the
+correct parser. See the [Parser](#parser) section for more information.
+
+[OpenTabletDriver#2433]: https://github.com/OpenTabletDriver/OpenTabletDriver/issues/2433
+
+#### Getting the LPMM
+
+This is almost always exactly 100 or exactly 200 for both axises.
+
+> If your tablet only offers an line per inch (LPI) value, you can divide the
+  LPI by `25.4` to get the LPMM.
+
+#### Calculating the width and height
+
+With the LPMM and maximum values reported by the tablet in hand, calculate the
+width and the height using the following formulas:
+
+```py
+Width = MaxX / Horizontal_LPMM
+Height = MaxY / Vertical_LPMM
+```
+
+Double check that the resulting width and height is close to the manufacturer's
+specified digitizer working area.
+
+A deviation of up to 10% is normal, but if it is obviously wrong (more than 50%
+out), confirm that both your LPMM and width/height values are correct.
+
+> If the guessed LPMM seems incorrect, it is better to manually measure the
+  width and the height of the digitizers active area.
+
+### Verify functionality
+
+Now that you've checked the values for correctness, you now have to validate
+the exposed functionality as much as possible.
+
+You don't need to validate every single feature set in the OS. For example,
+pressure is assumed working if the tablet debugger reports it correctly.
+
+You now need to double check that the following functionality completely works:
+
+- The entirety of the tablets area is usable
+- All pen buttons works and can be assigned
+- All auxiliary (tablet) buttons works and can be assigned
+- Extra tool functionality, if able:
+  - Tablet mouse
+  - Any other tools your tablet might support
+
+If anything is not working to your expectations, you might be using the wrong
+parser, or you specified some values incorrectly.
```

Comment:
Could we mention the Discord server directly here? If anybody is discouraged because they can't get it working a direct link to help will connect to them more than just the header at the top of the screen.